public/js/get_events.js

```diff
@@ -0,0 +1,22 @@
+/* global XMLHttpRequest */
+var module = (function () {
+  console.log(module);
+  return {
+    getEvents: function (path, callback) {
+      var baseUrl = 'https://nazareth-open-tourism-platform.herokuapp.com/events';
+      var url = baseUrl + path;
+      var xhr = new XMLHttpRequest();
+      xhr.onreadystatechange = function () {
+        if (xhr.readyState === 4 && xhr.status === 200) {
+          console.log('date from', xhr.responseText);
+          callback(null, JSON.parse(xhr.responseText));
+        } else if (xhr.status === 500) {
+          var errorMessage = new Error('Server Error! Status: ' + xhr.status);
+          callback(errorMessage, null);
+        }
+      };
+      xhr.open('GET', url, true);
+      xhr.send();
+    }
+  };
+})();
```

Comment:
this file doesn't need to be an IIFE, there is no gain from using one here. It should be
```js
var getEvents = function () {
...
}
```

Reply:
@mattlub whats the gain of removing it ?

Reply:
readibility, among other things

Reply:
im not sure how can it affect readability if anything it improves it cause you specify which module you are selecting and which function from it you are using (e.g: moduleA.functionB )

Reply:
If you have to make it into a 'module', please give the module a useful name, and write it like this:
```js
var betterName = {
  getEvents: ...
}
```
There is no need for an IIFE!!!

Reply:
@mattlub to me it looks like you want to change this cause you prefer it that way not because of bad practice or bad code. but i'll change it :+1: :maple_leaf: